Makefile

```diff
@@ -37,7 +37,20 @@ kolibri.apk: p4a_android_distro
 build_docker: Dockerfile
 	docker build -t android_kolibri .
 
+create_premigrated_db:
+	pip install whl/*.whl
+	KOLIBRI_HOME=tmphome kolibri manage migrate
+	cp tmphome/db.sqlite3 src/db.sqlite3.empty
```

Comment:
w00t!

Reply:
Just wait... it escalated from there! Additional PR incoming.